tools/codesigningtool/README

```diff
@@ -0,0 +1 @@
+codesigningtool selects the appropriate signing identity for iOS apps.
```

Comment:
`... for Apple executable bundles. codesigningtool uses codesign which is only available in macOS platforms.`

Reply:
👍